### DISCUSSION TITLE
site https://www.primefaces.org/designer-jsf/ response 503 Service Unavailable

### DISCUSSION THREAD

**Discussion by kaisernova** [primefaces/community#2424] [PrimeFaces] [answered]
I want to test the theme designer (https://www.primefaces.org/designer-jsf/), but everytime that I clicked in any option I got the following message (check dev tools -> network in browser):
`<html><head>
<title>503 Service Unavailable</title>
</head><body>
<h1>Service Unavailable</h1>
<p>The server is temporarily unable to service your
request due to maintenance downtime or capacity
problems. Please try again later.</p>
<hr>
<address>Apache/2.4.18 (Ubuntu) Server at www.primefaces.org Port 443</address>
</body></html>
`

![image](https://github.com/user-attachments/assets/dc6e933f-bf02-41bb-b987-50161b1be4ad)

**Comment by melloware** [comment, marked as answer]
Duplicate of https://github.com/orgs/primefaces/discussions/2408
